jbmc/src/java_bytecode/nondet.h

```diff
@@ -0,0 +1,28 @@
+/// Author: DiffBlue Limited. All Rights Reserved.
+
+#ifndef CPROVER_JAVA_BYTECODE_NONDET_H
+#define CPROVER_JAVA_BYTECODE_NONDET_H
+
+#include <util/std_code.h>
+#include <util/std_expr.h>
+#include <util/symbol_table.h>
+
+symbol_exprt generate_nondet_int(
+  int64_t min_value,
+  int64_t max_value,
+  const std::string &name_prefix,
+  const typet &int_type,
+  const source_locationt &source_location,
+  symbol_table_baset &symbol_table,
+  code_blockt &instructions);
+
+typedef std::vector<codet> alternate_casest;
```

Comment:
this typedef is not necessary (the semantic of the vector should be given in the variable name rather than the type)